Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/service/EventLogService.java

```diff
@@ -152,4 +153,12 @@ boolean saveFileToVolume(String filename, String prefix,
                            ByteArrayOutputStream baos) throws Exception;
 
   ConfigBean getConfigBean();
+
+
+  /**
+   * Find perspective tag that are linked to specified Log
+   * @param logId Log Id
+   * @return Perspective tags
+   */
+  List<String> getPerspectiveTagByLog(Integer logId) throws JsonProcessingException;
```

Comment:
Instead of JsonProcessingException which is an implementation detail (Jackson library), can we throw our domain-specific exception. At the client layer (Portal plugin), it only knows about domain excetion. For example, in this case, it makes sense to throw a UsermetadataException to represent any abnormal behavior in processing Usermetadata related: invalid dashboard json, invalid perspective tags, invalid filter json, etc.